apiserver/facades/agent/deployer/deployer_test.go

```diff
@@ -141,6 +136,17 @@ func (s *deployerSuite) SetUpTest(c *gc.C) {
 	)
 	c.Assert(err, jc.ErrorIsNil)
 	s.deployer = deployer
+
+	return ctrl
+}
+
+func (s *deployerSuite) SetUpTest(c *gc.C) {
+	s.ApiServerSuite.SetUpTest(c)
+
+	// The two known machines now contain the following units:
+	// machine 0 (not authorized): mysql/1 (principal1)
+	// machine 1 (authorized): mysql/0 (principal0), logging/0 (subordinate0)
+
```

Comment:
Move all of the state setup logic back here.

Reply:
I'll but it in a separate func.